misc/extra/src/test/java/org/openscience/cdk/validate/Geometry3DValidatorTest.java

```diff
@@ -44,5 +44,6 @@ void testEthane() throws Exception {
         ValidationReport report = engine.validateChemFile(chemFile);
         Assertions.assertEquals(0, report.getErrorCount());
         Assertions.assertEquals(0, report.getWarningCount());
+        reader.close();
```

Comment:
All ok but this could be better written as:

```java
try (MDLV2000Reader reader = new MDLV2000Reader(ins, Mode.STRICT)) {
    ChemFile chemFile = (ChemFile) reader.read((ChemObject) new ChemFile());
    ValidatorEngine engine = new ValidatorEngine();
    engine.addValidator(new Geometry3DValidator());
    ValidationReport report = engine.validateChemFile(chemFile);
    Assertions.assertEquals(0, report.getErrorCount());
    Assertions.assertEquals(0, report.getWarningCount());
}
```